src/test/java/foodwhere/model/stall/StallTest.java

```diff
@@ -94,4 +95,18 @@ public void equals() {
         editedAlice = new StallBuilder(TypicalStalls.ALICE).withReviews(TypicalReviews.BOB).build();
         assertFalse(alice.equals(editedAlice));
     }
+
+    @Test
+    public void getTagString_generalTesting_success() {
+        String testString = "test";
+        Stall testStallNoTag = new StallBuilder().withName(testString).withAddress(testString).build();
+        assertEquals("", testStallNoTag.getTagString());
+
+        Stall testStallWOneTag = new StallBuilder(testStallNoTag).withTags("tag").build();
+        assertEquals("tag", testStallWOneTag.getTagString());
+
+        String[] tags = new String[] {"tag", "tag2", "tag3"};
+        Stall testStallWManyTag = new StallBuilder(testStallNoTag).withTags(tags).build();
+        assertEquals("tag, tag2, tag3", testStallWManyTag.getTagString());
```

Comment:
This doesn't seem to be a good test since the order of the tags doesn't seem guaranteed by HashSet.

Reply:
Resolved by checking individual tags separated by `,` in a loop. This is possible as Tags are only alphanumerical in nature.